src/include/job.h

```diff
@@ -1061,7 +1061,9 @@ task_find	(job		*pjob,
 #define JOB_EXEC_HOOKERROR	-22 /* job exec failed due to */
 				    /* unexpected exception or */
 				    /* hook execution timed out */
-
+#define JOB_EXEC_UPDATE_ALPS_RESV_ID 1  /*update ALPS reservation ID to parent mom as soon as it is available.
+					  This is neither a success nor a failure exit code,
+					  so we are using a positive value*/
```

Comment:
Please use the appropriate multi-line comment format (with spaces).
```
/*
 * This is a multi-
 * line comment.
 */
```
You are welcome (and encouraged) to fix additional comments that do not conform.

Reply:
Done. Please have a look.